templates/tfengine/components/cicd/configs/tf-apply.yaml

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Note: Terraform version used in the automation is 0.14.7.
-{{- $cft := "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a9f8f41fdd1f2e1919a052df23d5a15b07ecd29bb447d5cd111d5a80228f1e75"}}
+# Note: Terraform version used in the automation is 0.14.8.
```

Comment:
also update the versions in README files for 3 binaries

Reply:
Done.